staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -329,10 +332,9 @@ func (r *crdHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	if utilfeature.DefaultFeatureGate.Enabled(features.WarningHeaders) {
```

Comment:
Why are we no longer feature gating this?

Reply:
moved to construction time, rather than request serve time

Reply:
> moved to construction time, rather than request serve time (`Move feature gate check to crdInfo construction time`)

ok